### ISSUE TITLE
NPE in ManageSecurityTokenFragment

### ISSUE THREAD

**Issue by dschuermann** [open-keychain/open-keychain#2168]
Going back and forth using the key creation wizard for security tokens:

```
09-25 10:35:44.755  21364    21364         AndroidRuntime  D  Shutting down VM
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  FATAL EXCEPTION: main
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  Process: org.sufficientlysecure.keychain.debug, PID: 21364
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  java.lang.NullPointerException: Attempt to invoke virtual method 'void org.sufficientlysecure.keychain.ui.widget.StatusIndicator.setDisplayedChild(org.sufficientlysecure.keychain.ui.widget.StatusIndicator$Status)' on a null object reference
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at org.sufficientlysecure.keychain.ui.token.ManageSecurityTokenFragment.statusLineError(ManageSecurityTokenFragment.java:221)
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at org.sufficientlysecure.keychain.ui.token.ManageSecurityTokenPresenter.continueSearchAfterError(ManageSecurityTokenPresenter.java:92)
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at org.sufficientlysecure.keychain.ui.token.ManageSecurityTokenPresenter.access$900(ManageSecurityTokenPresenter.java:44)
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at org.sufficientlysecure.keychain.ui.token.ManageSecurityTokenPresenter$2.onLoadFinished(ManageSecurityTokenPresenter.java:254)
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at org.sufficientlysecure.keychain.ui.token.ManageSecurityTokenPresenter$2.onLoadFinished(ManageSecurityTokenPresenter.java:204)
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at android.support.v4.app.LoaderManagerImpl$LoaderInfo.callOnLoadFinished(LoaderManager.java:476)
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at android.support.v4.app.LoaderManagerImpl$LoaderInfo.reportStart(LoaderManager.java:312)
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at android.support.v4.app.LoaderManagerImpl.doReportStart(LoaderManager.java:824)
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at android.support.v4.app.Fragment.performStart(Fragment.java:2122)
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1151)
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1295)
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at android.support.v4.app.BackStackRecord.popFromBackStack(BackStackRecord.java:1038)
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at android.support.v4.app.FragmentManagerImpl.popBackStackState(FragmentManager.java:1796)
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at android.support.v4.app.FragmentManagerImpl.popBackStackImmediate(FragmentManager.java:626)
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at org.sufficientlysecure.keychain.ui.CreateKeyActivity.onSecurityTokenPostExecute(CreateKeyActivity.java:156)
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at org.sufficientlysecure.keychain.ui.base.BaseSecurityTokenActivity$1.onPostExecute(BaseSecurityTokenActivity.java:163)
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at org.sufficientlysecure.keychain.ui.base.BaseSecurityTokenActivity$1.onPostExecute(BaseSecurityTokenActivity.java:136)
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at android.os.AsyncTask.finish(AsyncTask.java:695)
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at android.os.AsyncTask.-wrap1(Unknown Source:0)
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at android.os.AsyncTask$InternalHandler.handleMessage(AsyncTask.java:712)
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at android.os.Handler.dispatchMessage(Handler.java:105)
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at android.os.Looper.loop(Looper.java:164)
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at android.app.ActivityThread.main(ActivityThread.java:6541)
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at java.lang.reflect.Method.invoke(Native Method)
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
09-25 10:35:44.762  21364    21364         AndroidRuntime  E  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)
```

**Comment by dschuermann** [comment]
To reproduce:
1. hold card with key against it
2. select reset of card
3. use back button afterwards

**Comment by Valodim** [comment]
Doesn't reproduce for me with these steps. Do you mean press back when the "really reset?" dialog shows up?

**Comment by Valodim** [comment]
Probably fixed in 4a0c61a4782d947e1b9ee5c9431cc15051ab1b26
